File: LC1491.py
class Solution:
    def average(self, salary: List[int]) -> float:
        mi = inf
        ma = -inf
        sum = 0

        for i in salary:
            sum += i
            mi = min(mi, i)
            ma = max(ma, i)
        
        return (sum - mi - ma)/(len(salary)-2)

File: LC1491.cpp
class Solution {
public:
    double average(vector<int>& salary) {
        double sum = 0;
        int ma = INT_MIN, mi = INT_MAX;
        
        for(auto i : salary){
            if(i > ma)
                ma = i;
            if(i < mi)
                mi = i;
            sum += i;
        }

        return (double)(sum - ma - mi)/(double)(salary.size()-2);
    }
};
